### ISSUE TITLE
Migration to APIs.guru domain

### ISSUE THREAD

**Issue by IvanGoncharov** [APIs-guru/openapi-directory#85]
**TL;DR** If you using our API please change `https://apis-guru.github.io/api-models/api/v1/list.json` to `https://api.apis.guru/v2/list.json`

We plan to move APIs.guru out of the proof-of-concept stage. Also, we want to improve our visibility in search engines and GitHub search. This involves some breaking changes to better organize URLs/repos and give them meaningful names (api-models R.I.P. :cry:). So if you search for something like "Swagger API catalog" you should get APIs.guru at the first page.

 As part of this effort, **we are moving all our resources to the `APIs.guru` domain** (~~we will continue using GitHub pages under the hood~~).

Here is our migration plan:
  - [x] Provide the copy of resources on APIs.guru domain;
  - [x] Contact all potential API users on Twitter, Gitter, email, etc;
  - [x] **Old URLs will be removed on ~~22th Jun~~ 29th Jun**. If you need more time to do migration, please, comment on this issue;
  - [x] 404 page with migration instructions on it;
  - [x] Add Google Analytics to 404 page to track invalid requests and sites from which they are originated;

On 22th Jun all links to `apis-guru.github.io` will be automatically redirected to `apis.guru`.
Here is the table of **breaking chages** which helps you to easily migrate URLs:

| Name | Deprecated URL | New URL 
|:-------:|-------------------------|--------------
| API | apis-guru.github.io/api-models/api/v1/ | api.apis.guru/v2/
| Specs | apis-guru.github.io/api-models/ | api.apis.guru/v2/specs/
| Badges | apis-guru.github.io/api-models/banners/*_banners.svg | api.apis.guru/badges/*.svg
| Branding assets | apis-guru.github.io/api-models/branding/ | apis.guru/branding/

**All other URLs from `apis-guru.github.io` will be automatically redirected but we still recommend you to rebase them on `apis.guru`**

**Comment by IvanGoncharov** [comment]
We tried GitHub Pages + CloudFlare combo to serve API on `api.apis.guru` domain.
But create a lot of problems since CloudFlare build for hosting websites and block "suspicious" traffic.
Especially if it coming from headless browsers like PhantomJS.

So we decided to try S3 for API hosting and so far it looks good.
I will run few experiments with it on https://s3.amazonaws.com/api.apis.guru/v2/list.json
And if everything will work I will switch it to be the official backend for `api.apis.guru`.

**Comment by IvanGoncharov** [comment]
We setup new domain using S3 + CloudFront but we still need to setup custom SSL certificate in order to host it under `api.apis.guru`. It will take us couple mode days that mean we can't do migration today so we postpone it for one more week until 29th of June.

**Comment by IvanGoncharov** [comment]
We switched to the new backend so starting from now we have everything nicely hosted on AWS with our own SSL certificate.

**Comment by IvanGoncharov** [comment]
Few PRs is still open but we need to move forward and we will shut down old API on 8th Jul.
